### DISCUSSION TITLE
GitHub Container Registry (ghcr.io) public images not visible anonymously on the web

### DISCUSSION THREAD

**Discussion by dhirupandey** [community/community#26886] [Actions]
<p>public docker images on github container registry (<a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a>) not visible anonymously.</p>
<p>Unable to see the public images on the <a href="https://github.com/ORG/packages" rel="noopener nofollow ugc">https://github.com/ORG/packages</a> when I am not logged in (anonymous). If the images are public then they should be visible anonymously</p>
<p>One can pull the images anonymously though as long as they know the name of the image</p>

**Comment by airtower-luna** [comment]
<p>I’ve just confirmed this. My personal package page is empty when not logged in, and so is the repository packages page of a repository linked with an image. If I try to use a direct link to the container page (e.g. <a href="https://github.com/users/airtower-luna/packages/container/package/hello-ghcr-meow">my example container</a>) I get 404.</p>

**Comment by whitneyimura** [comment]
<p>:wave: The team is tracking this issue and working on an immediate fix.  Thank you for reporting!  We will give a status update once the fix is live.</p>

**Comment by clarkbw** [comment, marked as answer]
<p>Fix went out minutes ago. Let us know if you still see an issue!</p>

**Comment by dhirupandey** [comment]
<p>It works ! You guys totally rock ! Great job … Thanks</p>
